internal/app/decision_maker.go

```diff
@@ -87,10 +87,12 @@ func (cs *currentState) makePlan(s *state) *plan {
 
 // decide makes a decision about what commands (actions) need to be executed
 // to make a release section of the desired state come true.
-func (cs *currentState) decide(r *release, n *namespace, p *plan, c *chartInfo) {
+func (cs *currentState) decide(r *release, n *namespace, p *plan, c *chartInfo, settings config) {
```

Comment:
nit: do we really need to pass the whole settings here or should we just pass the skip ignore flag?, Not a big deal since this is a long running app, just thinking out loud.

Reply:
I did that assuming some more fields can be used from settings inside the function. Do you think it's valid assumption to leave it as I did it?

Reply:
Yeah, let's keep it as you have it here.